extensions/wikia/WikiFactory/WikiFactory.php

```diff
@@ -1166,36 +1166,6 @@ static public function IDtoDB( $city_id, $master = false ) {
 		return isset( $oRow->city_dbname ) ? $oRow->city_dbname : false;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the literal "false" boolean value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1125)

[project-id]: wikia:app